cmd/podman/images/save.go

```diff
@@ -83,9 +82,13 @@ func saveFlags(flags *pflag.FlagSet) {
 
 }
 
-func save(cmd *cobra.Command, args []string) error {
+func save(cmd *cobra.Command, args []string) (finalErr error) {
 	var (
-		tags []string
+		tags      []string
+		pipePath  string
```

Comment:
(Non-blocking: `pipePath`, `cleanup`, (`err`?) can be just initialized with `:=` on first use without a forward declaration I think.)